tcs-api/src/main/java/com/transformuk/hee/tis/tcs/api/dto/PostFundingDTO.java

```diff
@@ -19,6 +19,8 @@ public class PostFundingDTO implements Serializable {
 
   private String fundingType;
 
+  private String fundingSubType;
```

Comment:
I think we ususally use UUID type for uuids, eg. CurrculumDTO, ConditionsOfJoiningDto, etc.